polyfills/Intl/update.task.js

```diff
@@ -35,28 +46,29 @@ module.exports = function(grunt) {
 	// don't test every single locale - it will be too slow
 	configSource.test = { ci: false };
 
-	var configFileSource = JSON.stringify(configSource, null, 4);
+	const configFileSource = JSON.stringify(configSource, null, 4);
 
-	grunt.log.writeln('Importing Intl polyfill from ' + IntlSourcePath);
-
-	var IntlPolyfillSource = fs.readFileSync(IntlSourcePath);
-	grunt.file.write(path.join(IntlPolyfillOutput, 'polyfill.js'), IntlPolyfillSource);
+	console.log('Importing Intl.~locale.* polyfill from ' + LocalesPath);
+	const locales = fs.readdirSync(LocalesPath);
+	locales.forEach(function (file) {
+		const locale = file.slice(0, file.indexOf('.'));
+		const localeOutputPath = path.join(LocalesPolyfillOutput, locale);
 
+		if (!existsSync(localeOutputPath)) {
+			mkdirp.sync(localeOutputPath);
+		}
 
-	grunt.log.writeln('Importing Intl.~locale.* polyfill from ' + LocalesPath);
+		const localePolyfillSource = fs.readFileSync(path.join(LocalesPath, file));
+		const polyfillOutputPath = path.join(localeOutputPath, 'polyfill.js');
+		const detectOutputPath = path.join(localeOutputPath, 'detect.js');
+		const configOutputPath = path.join(localeOutputPath, 'config.json');
 
-	var locales = fs.readdirSync(LocalesPath);
-	locales.forEach(function (file) {
-		var locale = file.slice(0, file.indexOf('.'));
-		var localePolyfillSource = fs.readFileSync(path.join(LocalesPath, file));
-		// produce a new locale data polyfill
-		grunt.file.write(path.join(LocalesPolyfillOutput, locale, 'polyfill.js'), localePolyfillSource);
-		grunt.file.write(path.join(LocalesPolyfillOutput, locale, 'detect.js'), detectFileSource);
-		grunt.file.write(path.join(LocalesPolyfillOutput, locale, 'config.json'), configFileSource);
+		writeFileIfChanged(polyfillOutputPath, localePolyfillSource);
+		writeFileIfChanged(detectOutputPath, detectFileSource);
+		writeFileIfChanged(configOutputPath, configFileSource);
 	});
 
 
-	grunt.log.writeln(locales.length + ' imported locales');
-	grunt.log.writeln('Intl polyfill imported successfully');
-
+	console.log(locales.length + ' imported locales');
+	console.log('Intl polyfill imported successfully');
 };
```

Comment:
Not quite, it now won't modify any files which have not changed, this is a good thing as before git would think every single file has been changed when in fact they had not.